examples/hosts/app-integration/external-data/README.md

```diff
@@ -6,15 +6,54 @@ Please note that the ideas explored here are experimental and under development.
 
 ## Scenario
```

Comment:
Be sure to run `npm run format` from within the package. I suspect prettier will want to upate some formatting here :)